graphql/resolvers.go

```diff
@@ -2419,10 +2428,17 @@ func (r *taskResolver) Annotation(ctx context.Context, obj *restModel.APITask) (
 		return nil, nil
 	}
 	apiAnnotation := restModel.APITaskAnnotationBuildFromService(*annotation)
+	return apiAnnotation, catcher.Resolve()
+}
 
-	//todo: get jira ticket objects before returning
+func (r *annotationResolver) Issues(ctx context.Context, obj *restModel.APITaskAnnotation) ([]*restModel.APIIssueLink, error) {
+	issues, err := restModel.GetJiraTickets(obj.Issues)
+	return issues, err
+}
```

Comment:
can return restModel.GetJiraTickets(obj.Issues) directly since you aren't using the variables (same with SuspectedIssues)